onebusaway-android/src/main/java/org/onebusaway/android/ui/ReportProblemFragmentBase.java

```diff
@@ -116,6 +127,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    /**
+     * Validates report arguments before submitting a problem
+     *
+     * @return false if the report is not valid
+     */
+    protected boolean isReportArgumentsValid() {
+        //Check if a user selects a problem category
```

Comment:
@cagryInside Could you please change the comments to the format of `// Check...`?  There should be a space between the double slash and the first word.  I've been guilty of not formatting this correctly in the past, and I've been trying to change to the correct format as I touch the code.